.github/actions/templates/avm-validateModuleDeployment/action.yml

```
@@ -290,7 +291,8 @@ runs:
 
           if($moduleTemplatePossibleParameters -contains 'resourceLocation') {
             $functionInput.AdditionalParameters += @{
-              resourceLocation = '${{ steps.get-resource-location.outputs.resourceLocation }}'
+              # resourceLocation = '${{ steps.get-resource-location.outputs.resourceLocation }}'
+              resourceLocation = 'eastus'
```

```suggestion
              resourceLocation = '${{ steps.get-resource-location.outputs.resourceLocation }}'
```
once done with the PR...